rules/LinearWaterway.js

```diff
@@ -8,6 +8,8 @@ exports.candidateDistanceSigma = 1.0; // 1.0 * (CE95 + Worst CE95);
 exports.matchThreshold = parseFloat(hoot.get("waterway.match.threshold"));
 exports.missThreshold = parseFloat(hoot.get("waterway.miss.threshold"));
 exports.reviewThreshold = parseFloat(hoot.get("waterway.review.threshold"));
+// See #3047
+exports.matchCandidateCriterion = "hoot::LinearWaterwayCriterion";
```

Comment:
After #3047 you won't have to specify both exports.matchCandidateCriterion and isMatchCandidate.  You'd pick one or the other.